subprojects/docs/src/snippets/tasks/incrementalBuild-customTaskClass/groovy/build.gradle

```diff
@@ -87,7 +101,7 @@ tasks.register('processTemplatesWithExtraInputs', ProcessTemplates) {
 
 // tag::inferred-task-dep-via-outputs[]
 tasks.register('packageFiles', Zip) {
-    from processTemplates.map {it.outputs }
+    from processTemplates.map { it.outputDir }
```

Comment:
This change was made to reference a supported serializable object, maintaining the implicit dependency between tasks.